storybook/webpack/elm-webpack-loader-fix.js

```diff
@@ -7,24 +7,24 @@
 
 "use strict"
 
-var fs = require("fs")
-var path = require("path")
-var loaderUtils = require("loader-utils")
-var elmCompiler = require("node-elm-compiler")
-var yargs = require("yargs")
+const fs = require("fs")
+const path = require("path")
+const loaderUtils = require("loader-utils")
+const elmCompiler = require("node-elm-compiler")
+const yargs = require("yargs")
 
-var runningInstances = 0
-var alreadyCompiledFiles = []
+let runningInstances = 0
+const alreadyCompiledFiles = []
 
-var defaultOptions = {
+const defaultOptions = {
   cache: false,
   forceWatch: false,
-  optimize: false,
+  optimize: true,
```

Comment:
I'm curious if this will actually work. There was a reason we didn't turn `optimize: true` on when we upgraded to Elm 0.19 but that was 3 years ago and it escapes me now...

If it looks fine in storybook though it's probably safe to merge - these settings won't effect any consuming repos